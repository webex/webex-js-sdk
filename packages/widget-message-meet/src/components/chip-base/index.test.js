import React from 'react';
import renderer from 'react-test-renderer';

import ChipBase from '.';

describe(`ChipBase component`, () => {
  let component;
  const onRemove = jest.fn();
  const props = {
    children: <div />,
    id: `testFile`,
    onRemove
  };

  it(`renders properly`, () => {
    component = renderer.create(<ChipBase {...props} />);
    expect(component).toMatchSnapshot();
  });

});
