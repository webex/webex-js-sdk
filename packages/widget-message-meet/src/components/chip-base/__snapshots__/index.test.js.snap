exports[`ChipBase component renders properly 1`] = `
<div
  className="chip">
  <div />
  <div
    className="chip-action">
    <button
      className="button"
      onClick={[Function]}
      onKeyUp={[Function]}>
      <div
        className="icon"
        title={undefined} />
    </button>
  </div>
</div>
`;
